.github/workflows/publish-npm.yml

```diff
@@ -88,16 +88,9 @@ jobs:
         run: |
           npm publish
 
-      - uses: montudor/action-zip@v1
-        if: ${{ inputs.github_release == 'true' }}
+      - uses: uniquesca/continuous-integration/publish-github-release@feature/publish-github-release
```

Comment:
This PR is ready to be accepted. Clean it up to use proper triggers here (and check if everything else is ready)